src/utils/streakUtils.js

```
@@ -0,0 +1,68 @@
+  ];
+
+  let currentStreak = 0;
+  const today = new Date();
+  let currentDate = new Date(today.toLocaleDateString('en-CA', { timeZone }));
+
+  const dayFormatter = new Intl.DateTimeFormat('en-US', {
+    timeZone,
+    weekday: 'short',
+  });
+
+  for (let i = 0; i < uniqueCompletionDates.length; i++) {
```

ESLint

[error] Unary operator '++' used.

(no-plusplus)